homeassistant/components/openweathermap/weather_update_coordinator.py

```diff
@@ -46,9 +46,10 @@
     FORECAST_MODE_ONECALL_DAILY,
     FORECAST_MODE_ONECALL_HOURLY,
     WEATHER_CODE_SUNNY_OR_CLEAR_NIGHT,
-    TEMP_CELSIUS,
 )
 
+from homeassistant.const import TEMP_CELSIUS
```

Comment:
We'll get a linter error here since the imports aren't sorted. This import should be grouped with the other homeassistant imports and sorted :abcd:. If you have `isort` you can run that and it will do it for you.